boards/shields/ls0xx_generic/doc/index.rst

```diff
@@ -0,0 +1,124 @@
+.. _ls0xx_generic_shield:
+
+Sharp memory display generic shield
+###################################
+
+Overview
+********
+
+This is a generic shield for Sharp memory pixel LCD. It supports
+displays of LS0XX type. These displays have an SPI interface and
+few other pins. Note that the SCS is active high for this display.
+
+The DISP pin controls whether to display memory
+contents or show all white pixels, this can be connected
+directly to VDD, to always display memory contents or connected
+to a gpio. If devicetree contains ``disp-en-gpios`` then it will be set to
+high during driver initialization. Display blanking apis can be used
+to control it.
+
+Sharp memory displays require toggling the VCOM signal periodically
+to prevent a DC bias ocurring in the panel as mentioned in the `appnote`_
+and `datasheet`_. The DC bias can damage the LCD and reduce the life.
+This signal must be supplied from either serial input (sw) or an external
+signal on the EXTCOMIN pin. The driver can do this internally
+``CONFIG_LS0XX_VCOM_DRIVER=y`` (default) else user can handle it in
+application code ``CONFIG_LS0XX_VCOM_DRIVER=n``.
+
+CONFIG_LS0XX_VCOM_DRIVER=y
+  Driver will handle VCOM toggling. User can control method of toggling
+  by defining ``extcomin-gpios`` in the device tree.
+
+extcomin-gpios is not defined
+  VCOM is toggled through serial input by software.
+  EXTMODE pin is connected to VSS
+  Important: User has to make sure `display_write` is
+  called periodically for toggling VCOM.
+  If there is no data to update then buf can be set to NULL then
+  only VCOM will be toggled.
+  If using lvgl and there is no data to update ``lv_task_handler``
+  will not call `display_write`. So explicit call to `display_write`
```

Comment:
I do not think it is easy to achieve. The driver is not mutually exclusive in this regard, is it?